generateExpressionTests.py

```diff
@@ -0,0 +1,291 @@
+import re
+import os
+import sys
+import subprocess
+from argparse import ArgumentParser, RawTextHelpFormatter
+
+if os.name == "nt":  # Windows
+    make = "mingw32-make"
+else:
+    make = "make"
+
+src_folder = "./src/test/expressions/"
+build_folder = "./test/expressions/"
+exceptions_list_location = (
+    "./lib/stan_math/test/expressions/stan_math_sigs_exceptions.expected"
```

Comment:
It wil be more conveninet to remove exceptions if they are in math. If they were in this repo we would need to mirror all generalize PRs in this repo to remove exceptions from the list.

Reply:
This pull request came up in the Math meeting. Can these tests live in math? @SteveBronder and @syclik didn't think the dependency this would introduce between Math and Stanc3 was a big deal.